lib/http1.c

```diff
@@ -58,7 +58,7 @@ struct st_h2o_http1_conn_t {
     h2o_timer_t _timeout_entry;
     uint64_t _req_index;
     size_t _prevreqlen;
-    size_t _reqsize;
+    size_t _headers_size;
```

Comment:
I would appreciate it if you could either change the name of the variable to something more appropriate, or add a comment explaining how the value is used.

This is because the value does not always represent the size of the header fields with the proposed change (when the request has a body, it becomes zero). It is my understanding that the amount of data left un-consumed in the socket read buffer.